DataModelHelloWorld/Cpp/SimpleIntroClientLibrary/SimpleIntroClientLibrary.vcxproj

```diff
@@ -172,20 +176,38 @@
     <ClCompile Include="SimpleIntroExtension.cpp" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="amd64\dbgcore.dll" />
+    <None Include="amd64\dbgeng.dll" />
+    <None Include="amd64\dbghelp.dll" />
+    <None Include="amd64\dbgmodel.dll" />
+    <None Include="arm64\dbgcore.dll" />
+    <None Include="arm64\dbgeng.dll" />
+    <None Include="arm64\dbghelp.dll" />
+    <None Include="arm64\dbgmodel.dll" />
     <None Include="packages.config" />
     <None Include="SimpleIntro.def" />
+    <None Include="woa\dbgcore.dll" />

```

Comment:
>woa [](http://example.com/codeflow?start=19&length=3)

ARM32 is... on its way out. Make me question the wisdom of this.

Reply:
This is automatic from the Microsoft.Debugging.Platform.DbgEng NuGET package.  WoA will have to be removed from some version of that NuGET before it'll get removed here.